packages/react/src/VideoPlayer/components/VideoTooltip/VideoTooltip.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import clsx from 'clsx'
+
+import {Text} from '../../../Text'
+import styles from '../../VideoPlayer.module.css'
+
+type VideoTooltipProps = React.HTMLAttributes<HTMLDivElement>
+
+export const VideoTooltip = ({children, className, ...rest}: VideoTooltipProps) => (
+  <div className={clsx(styles.VideoPlayer__tooltip, className)} aria-hidden="true" {...rest}>
```

Comment:
Should this be `aria-hidden`?

Reply:
Hopefully we can ditch this entirely and just use `Tooltip`, in which case this file can be deleted

Reply:
I don't know whether this will have had sign-off previously? I don't think I've really modified this file, just moved it.

Reply:
> I don't know whether this will have had sign-off previously?

I don't think it did. If you're keeping the custom tooltip though, are you still removing the aria-hidden? I wasn't sure what it was doing or why it was needed.

Reply:
Cool, thanks for the info. Yes it's removed 👍